src/test/java/org/jenkinsci/plugins/ownership/model/runs/RunOwnershipActionTest.java

```diff
@@ -51,6 +55,27 @@
     public JenkinsRule jenkinsRule = new JenkinsRule();
     
     @Test
+    @Bug(28881)
```

Comment:
It will only happen if I bump the core dependency or explicitly add the annotation dep